src/Asv.Drones.Gui/Shell/Pages/PacketViewer/PacketViewerViewModel.cs

```diff
@@ -75,9 +74,15 @@ public PacketViewerViewModel(IMavlinkDevicesService mavlinkDevicesService, IAppl
         Icon = MaterialIconKind.Package;
         ExportToCsv = ReactiveCommand.CreateFromTask(Export, this.WhenValueChanged(_ => _.IsPause));
         ClearAll = ReactiveCommand.Create(() => _packetsSource.Clear());
-
-        _converters = converters.OrderBy(_ => _.Order);
-
+        _printer = new PacketPrinter(new List<IPacketPrinterHandler>()
```

Comment:
The purpose of PacketPrinter is to ensure extensibility through plugins when needed. The IPacketPrinterHandler must be injected into the constructor via the IoC container.